Unified/checkor.py

```diff
@@ -1012,10 +1013,43 @@ def upward( ns ):
 
         time_point("checked custodiality", sub_lap=True)
 
-        ## presence in phedex
+        ## presence in phedex and/or rucio
         phedex_presence ={}
+        rucioClient = RucioClient()
         for output in wfi.request['OutputDatasets']:
-            phedex_presence[output] = phedexClient.getFileCountDataset(url, output )
+            _,dsn,process_string,tier = output.split('/')
+            if tier in set(UC.get('tiers_to_rucio_relval')) | set(UC.get('tiers_to_rucio_nonrelval')):
```

Comment:
First question I had was: thus the precedence work as we expect? So, to be on the safe side, we could do:
`if tier in set(UC.get('tiers_to_rucio_relval')) or tier in set(UC.get('tiers_to_rucio_nonrelval')):`

Reply:
Hi @amaltaro , I just changed the configuration flags to exactly those names and made a union of the two sets. 

@sharad1126 Please mind that, now since I have split the configuration flags and reused the names from this PR [1], I no longer have them in the current PR and so the former needs to be merged before this one. Thanks!  

[1]
https://github.com/CMSCompOps/WmAgentScripts/pull/532/commits/ba6634c1ae4c18b234c660ab0ce18f23a90a088e#diff-d4ad4f9352ebe88910fb865bcdc6703dR233-R240

Reply:
@todor-ivanov the #532 is ready to be merged from my side.

Reply:
@amaltaro if you are ok with the changes done, should we resolve this conversation?

Reply:
yes, doing so